schema/article.js

```diff
@@ -24,6 +25,10 @@ let ArticleType = new GraphQLObjectType({
     author: {
       type: AuthorType,
       resolve: ({ author }) => author
+    },
+    thumbnail_image: {
+      type: Image.type,
+      resolve: ({ thumbnail_image }) => thumbnail_image
```

Comment:
oh interesting, i hadn't considered that. We have thumbnail images and titles as their own fields in the API, but yea no reason why they couldn't be combined here. But also, no reason why they need to be combined here either?